samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/Startup.cs

```diff
@@ -119,7 +120,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<BotAdapter>(sp => sp.GetService<BotFrameworkHttpAdapter>());
 
             // Register the skills conversation ID factory, the client and the request handler.
-            services.AddSingleton<SkillConversationIdFactoryBase, SkillConversationIdFactory>();
+            services.AddSingleton<SkillConversationIdFactoryBase, Microsoft.Bot.Builder.Skills.SkillConversationIdFactory>();
```

Comment:
```suggestion
            services.AddSingleton<SkillConversationIdFactoryBase, SkillConversationIdFactory>();
```
As we removed the one of the project, it's not necessary to keep that reference